packages/cli/generation/remote-generation/remote-workspace-runner/src/publishDocs.ts

```diff
@@ -169,6 +169,35 @@ export async function publishDocs({
                 definitionV2: undefined
             });
 
+            if (response.ok) {
+                context.logger.debug(`Registered API Definition ${response.body.apiDefinitionId}`);
+                return response.body.apiDefinitionId;
+            } else {
+                switch (response.error.error) {
+                    case "UnauthorizedError":
+                    case "UserNotInOrgError": {
+                        return context.failAndThrow(
+                            "You do not have permissions to register the docs. Reach out to support@buildwithfern.com"
+                        );
+                    }
+                    default:
+                        if (apiName != null) {
+                            return context.failAndThrow(`Failed to register API ${apiName}`, response.error);
+                        } else {
+                            return context.failAndThrow("Failed to register API", response.error);
+                        }
+                }
+            }
+        },
+        async ({ api, apiName }) => {
+            context.logger.debug("Calling registerAPI with v2 definition... ", JSON.stringify(api, undefined, 4));
```

Comment:
lets actually delete this? i would love to delete all the code which prints out json payloads in the CLI since it looks a bit unprofessional